### ISSUE TITLE
Ionic v7.0.0-beta.2 getInitialPushPayload always returns null on iOS

### ISSUE THREAD

**Issue by dalobstah** [andrehtissot/cordova-plugin-fcm-with-dependecy-updated#101]
Ionic Info:
```
   Ionic CLI                     : 6.10.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.2.1
   @angular-devkit/build-angular : 0.901.8
   @angular-devkit/schematics    : 9.1.8
   @angular/cli                  : 9.1.8
   @ionic/angular-toolkit        : 2.2.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 5 other plugins)

Utility:

   cordova-res                          : not installed
   native-run (update available: 1.0.0) : 0.2.9

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v10.17.0 (/usr/local/Cellar/node@10/10.17.0/bin/node)
   npm        : 6.11.3
   OS         : macOS Catalina
   Xcode      : Xcode 11.5 Build version 11E608c
```

```
Device tested: iPhone 11 Pro Max
iOS version: 13.5
```

getInitialPushPayload is always null, even when app is opened from a notification. Tested in fresh project.

**Comment by algoj** [comment]
Hi, I am also encountering this problem. Any news if you already have a work around?

**Comment by medeirosrafael** [comment]
i was stucked in this problem..
The error is exactly here:  https://github.com/andrehtissot/cordova-plugin-fcm-with-dependecy-updated/blob/master/src/ios/FCMPlugin.m#L149

Trying transform NSData* into NSDictonary* returns nil..
Payload is ok in this point..
after several hours debugging and trying to learn a little objective-c language to fix.. I found this solution:

```objective-c
     NSDictionary *payloadDictionary = nil;
     NSError* error = nil;
     NSString *strISOLatin = [[NSString alloc] initWithData:payload encoding:NSISOLatin1StringEncoding];
     NSData *dataUTF8 = [strISOLatin dataUsingEncoding:NSUTF8StringEncoding];
     payloadDictionary = [NSJSONSerialization JSONObjectWithData:dataUTF8 options:0 error:&error];
      if (payloadDictionary != nil) {
         NSLog(@"Dict: %@", payloadDictionary);
      } else {
         NSLog(@"Error: %@", error);
      }
```

Based on this stackoverflow solution: https://stackoverflow.com/questions/48661465/converting-nsdata-to-nsdictionary

I'm not objc programmer, so, I don't know it's all right, but, it's working..

**Comment by andrehtissot** [comment]
Thank you @medeirosrafael.
The fix was released as 7.0.9.
